CTP/conf/isolation.conf

```diff
@@ -22,58 +22,42 @@
 #WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
 #USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# To configure host ip, port, user, password for instance01 test node to use ssh protocol
-env.instance01.ssh.host=<ip>
-env.instance01.ssh.port=22
-env.instance01.ssh.user=<user>
-env.instance01.ssh.pwd=<password>
-
-# The %query_editor port on instance1 test node
-env.instance01.broker1.BROKER_PORT=30093
-
-# The %BROKER1 port on instance1 test node
-env.instance01.broker2.BROKER_PORT=30094
-
-# To configure the second test node
-env.instance02.ssh.host=<ip>
-env.instance02.ssh.port=22
-env.instance02.ssh.user=<user>
-env.instance02.ssh.pwd=<password>
-env.instance02.broker1.BROKER_PORT=30093
-env.instance02.broker2.BROKER_PORT=30094
-
-# If execute continue testing when testing is interrupted
-main.mode.continue=false
-
-# Url of build which you will use to test
+# Define test instances for isolation - the host ip, password, port and user for ssh connection.
+# the prefix and suffix will be used as keywords for CTP configuration, and the second section - 'instance1' can be changed, and
+# it will also be logged into file as reference information. For multiple sets of test node environment, the instance names must be different
+env.instance1.ssh.host=<ip>
+env.instance1.ssh.port=22
+env.instance1.ssh.user=<user>
+env.instance1.ssh.pwd=<password>
+
+# Define %query_editor port for instance1 test node to avoid port conflicts
+env.instance1.broker1.BROKER_PORT=30093
+
+# Define the %BROKER1 port for instance1 test node to avoid port conflicts
+env.instance1.broker2.BROKER_PORT=30094
+
+# Define TCP port id for the CUBRID programs for instance1 to avoid port conflicts
+env.instance1.cubrid.cubrid_port_id=1524
+
+# Define the second test node
+env.instance2.ssh.host=<ip>
+env.instance2.ssh.port=22
+env.instance2.ssh.user=<user>
+env.instance2.ssh.pwd=<password>
+env.instance2.broker1.BROKER_PORT=30095
+env.instance2.broker2.BROKER_PORT=30096
+env.instance2.cubrid.cubrid_port_id=1525
+
+
+# Define the URL of build which will be used to test. If this parameter is not set or commented out, CTP will execute testing without build installation
 main.testbuild.url=http://127.0.0.1/daily_build/10.1.0.6928-58b5a97/drop/CUBRID-10.1.0.6928-58b5a97-Linux.x86_64.sh
 
-# The root path of test case, it should be checked out in each test node
+# Define the path of test cases used for testing, it should be checked out on test node, and CTP will get the total case list from the first test node.
```

Comment:
It's better to remove "and CTP will get the total case list from the first test node"